Example/iOS-Example/ViewController.swift

```diff
@@ -13,7 +13,7 @@ import FetchRequests
 class ViewController: UITableViewController {
     private(set) lazy var controller: FetchedResultsController<Model> = {
         let controller: FetchedResultsController<Model> = FetchedResultsController(
-            request: Model.fetchRequest(),
+            fetchDefinition: Model.fetchDefinition(),
```

Comment:
Would making the signature `definition` instead of `fetchDefinition` be cleaner? It's in the context of the `FetchedResultsController`, so I think the fetch can be implied.